scipy/stats/tests/test_distributions.py

```diff
@@ -1045,6 +1055,14 @@ def test_fit_warnings(self):
                       fscale=3)
 
 
+def _assert_lessthan_loglike(dist, data, func, **kwds):
```

Comment:
Wasn't this moved here in a previous PR or is that also pending?

Reply:
It is also pending in other PRs, that is correct.

Reply:
Thanks for the reminder @sethtroisi - 

In this case, where I already merged scipy/master with this PR, what is the best method to go back and remove this addition from the commit it is in?

In a testing duplicate branch, it appears that when I go back and edit that commit in interactive rebase that it co-authors me on every commit including those that were added from the merge. 

So instead, I am thinking of
1. doing a hard reset to prior to the master branch merge
2. interactive rebase to amend the commit where this code was added
3. merge scipy/master back into the branch
4. force push to update 

Feels a bit cumbersome, so before I do any of that, is there a more efficient way to do it that you had in mind?

Reply:
>In this case, where I already merged scipy/master with this PR

If this means you locally merged scipy/master locally since gh-12457 merged, then you have two copies of `_assert_lessthan_loglike` in the file, right? If that's the case, just delete one, commit, and push normally.

Reply:
You're right – I was thinking about removing the commit it came from, but that is much simpler.